Vungle Sample App Swift/Vungle Sample App Swift/ViewController.swift

```diff
@@ -30,9 +30,7 @@ class ViewController: UIViewController {
     @IBOutlet weak var playButton3: UIButton!
     @IBOutlet weak var checkCurrentStatusButton: UIButton!
     
-    var sdk:VungleSDK!
-    var placementIDsArray:Array<String>!
-    
+    var sdk:VungleSDK?
```

Comment:
In general we want to shy away from using `!`. It removes the safety of swift

Reply:
👍🏼